third-party/eazy/v1/openapi.yaml

```
@@ -1567,13 +1567,13 @@ paths:
         - in: path
           name: id
           required: true
-          description: ID of the node
+          description: The ID of the node
```

```suggestion
          description: The note's ID
```
Suggest using note here :)